src/customizations/volto/components/manage/Workflow/Workflow.jsx

```diff
@@ -231,10 +231,6 @@ class Workflow extends Component {
     selectedOption: this.props.currentStateValue,
   };
 
-  componentDidMount() {
```

Comment:
The workflow state doesn't seem to update anymore in the dropdown. While in the toolbar it says marked for deletion, in the popup state says Published. See bellow screenshot:

![Screenshot from 2024-04-16 17-16-50](https://github.com/eea/volto-eea-website-policy/assets/323385/7328e6ef-82d6-434d-b807-1d21bdecdf6d)